contrib/pipelines/secadvisor/Makefile

```diff
@@ -0,0 +1,21 @@
+PROG := secadvisor
+
+SRCS := \
+	main.go \
+	transform.go \
+
+.PHONY: all
+all:
+	go build -o $(PROG) $(SRCS)
+
+.PHONY: clean
+clean:
+	rm -rf $(PROG)
+
+.PHONY: install
+install:
+	sudo cp $(PROG).yml.default /etc/skydive/$(PROG).yml
```

Comment:
Why we copy to /etc/... location ... we should use the yml from command line ... why we need this 'global' thing ?

Reply:
fixed as above